identity/src/keypair.rs

```diff
@@ -264,6 +264,38 @@ impl Keypair {
     }
 }
 
+#[cfg(feature = "ecdsa")]
+impl From<ecdsa::Keypair> for Keypair {
+    fn from(kp: ecdsa::Keypair) -> Self {
+        #[allow(deprecated)]
+        Keypair::Ecdsa(kp)
```

Comment:
How do we implement the `From` once we removed the deprecated `Keypair::Ecdsa` @thomaseizinger?

Reply:
What do you mean? ECDSA is not deprecated, only the direct access to the enum variant is.

Once the type is opaque, the enum will still exist, just not publicly accessible and so we can still implement the `From` trait.

Reply:
Makes sense. Thanks. Feel free to go ahead with the release @thomaseizinger.